src/components/victory-animation.jsx

```diff
@@ -1,10 +1,11 @@
-/*global requestAnimationFrame, cancelAnimationFrame, setTimeout, clearTimeout*/
-
 import React from "react";
 import d3Ease from "d3-ease";
 import d3Interpolate from "d3-interpolate";
+import { timer } from "d3-timer";
 import { addVictoryInterpolator } from "../util";
 
+const VELOCITY_MULTIPLIER = 16.5;
```

Comment:
A comment for why this magic number is the value that it is seems of value to a newbie such as myself